packages/wallet-adapter/base/src/adapter.ts

```diff
@@ -130,8 +130,8 @@ export class StandardWalletAdapter extends BaseWalletAdapter implements Standard
         return this.#connect({ silent: true });
     }
 
-    async connect(): Promise<void> {
-        return this.#connect();
+    async connect(input?: StandardConnectInput): Promise<void> {
```

Comment:
This method is part of the Wallet Adapter interface, which is unaware of the Wallet Standard. The `autoConnect` method above serves the function of connecting with the `silent` flag.